ComponentKit/Core/CKComponentController.h

```diff
@@ -14,7 +14,7 @@
 
 @class CKComponent;
 
-@interface CKComponentController : NSObject
+@interface CKComponentController<__covariant ComponentType:CKComponent *> : NSObject
```

Comment:
Perfect, so ComponentType is defined as a covariant value with superclass CKComponent.